packages/template-retail-react-app/app/commerce-api/hooks/useBasket.js

```diff
@@ -174,6 +175,10 @@ export default function useBasket(opts = {}) {
                     return
                 }
 
+                if (!basket) {
+                    return
+                }
```

Comment:
```suggestion
             if (!id || !basket) {
                    return
                }
```

Reply:
Same as above.
Reverted useBasket back to original code.